python/python_direct/enum.cpp

```diff
@@ -93,6 +94,33 @@ void bindEnums(py::module& nvfuser) {
       .value("broadcast", IdMappingMode::BROADCAST)
       .value("permissive", IdMappingMode::PERMISSIVE)
       .value("loop", IdMappingMode::LOOP);
+
+  py::enum_<MatmulParams::TilingStrategy> tiling_strategy(
+      nvfuser, "MatmulTilingStrategy", py::module_local());
+  tiling_strategy.value(
+      "one_tile_per_cta", MatmulParams::TilingStrategy::OneTilePerCTA);
+  tiling_strategy.value(
+      "distribute_tiles_across_sms",
+      MatmulParams::TilingStrategy::DistributeTilesAcrossSMs);
+  tiling_strategy.value(
+      "distribute_stages_across_sms",
+      MatmulParams::TilingStrategy::DistributeStagesAcrossSMs);
```

Comment:
Any reason to prefer this style over the previous?
```suggestion

  py::enum_<MatmulParams::TilingStrategy>(nvfuser, "MatmulTilingStrategy", py::module_local())
      .value("one_tile_per_cta", MatmulParams::TilingStrategy::OneTilePerCTA)
      .value("distribute_tiles_across_sms", MatmulParams::TilingStrategy::DistributeTilesAcrossSMs)
      .value("distribute_stages_across_sms", MatmulParams::TilingStrategy::DistributeStagesAcrossSMs);
```

Reply:
I went back to the old style for heuristic params. The "chain" style can become unreadable if it gets too long. See python_frontend/python_bindings.cpp.